app/views/posts/_poll_answers_form.html.erb

```diff
@@ -1,7 +1,19 @@
 <%if account_signed_in? && answered(post.id)%>
-<p class="col-12 "><b>The poll has been already answered!</b></p>
+<% poll_total_count=get_poll_answer_values(post.id)%>
+<div class="card">
+<div class="card-header">
+<%= poll_total_count[0] %> Votes
+  </div>
+  <div class="card-body">
+    <p class="<%= poll_option_design(post.option1) %>"><b><%= poll_option_count(post.option1)%></b><%="  "%><%= post.option1%></p>
+    <p class="<%= poll_option_design(post.option2) %>"><b><%= poll_option_count(post.option2)%></b><%="  "%><%= post.option2%></p>
+    <p class="<%= poll_option_design(post.option3) %>"><b><%= poll_option_count(post.option3)%></b><%="  "%><%= post.option3%></p>
+    <p class="<%= poll_option_design(post.option4) %>"><b><%= poll_option_count(post.option4)%></b><%="  "%><%= post.option4%></p>
+  </div>
```

Comment:
can you refactor this code please? all the lines are duplicated @SrinandhiniM

Reply:
Code refactored now